x/evm/types/codec.go

```diff
@@ -2,15 +2,16 @@ package types
 
 import "github.com/cosmos/cosmos-sdk/codec"
 
-var msgCodec = codec.New()
+// TODO: Is this the MAIN codec? or should it only be applied to eth specific messages (originally `MsgCodec`)
```

Comment:
I'll remove the comment but definitely something to keep in mind. I think it will be quite obvious if we are using the wrong codec, however I do suspect we need Amino somewhere